providers/mailgun.go

```diff
@@ -0,0 +1,23 @@
+package providers
+
+const Mailgun Provider = "mailgun"
+
+func init() {
+	SetInfo(Mailgun, ProviderInfo{
+		AuthType: Basic,
+		BaseURL:  "https://api.mailgun.net/",
```

Comment:
Please remove the trailing `/` - that will cause calls like `proxy.withampersand.com/foo` to resolve to `https://api.mailgun.net//foo`

Reply:
amended